hal_ugv/Control control
---
# RESULT
bool    success
string  status
